scripts/nx-release.ts

```diff
@@ -241,7 +239,16 @@ function parseArgs() {
       description:
         'The version to publish. This does not need to be passed and can be inferred.',
       default: 'minor',
-      coerce: (version) => {
+      coerce: (version: string) => {
+        const isGithubActions = !!process.env.GITHUB_ACTIONS;
```

Comment:
Added extra safety check for the combination that bit us yesterday - if in the context of GitHub Actions we should always be receiving an explicit version number, not a relative keyword (our default of `minor`)